cmd/server/main.go

```diff
@@ -20,10 +20,17 @@ import (
 )
 
 const (
-	accessURL    = "access-001.devnet49.nodes.onflow.org:9000"
-	coinbaseAddr = "0xf02c1c8e6114b1dbe8937a39260b5b0a374432bb"
+	defaultAccessURL = grpc.EmulatorHost
+	coinbaseAddr     = "0xf02c1c8e6114b1dbe8937a39260b5b0a374432bb"
 )
 
+// TODO(m-Peter): These should be updates to the EVM location,
+// as soon as it gets merged.
+var evmEventTypes = []string{
+	"flow.evm.BlockExecuted",
```

Comment:
how about using:
```go
types.EVMLocation{}.TypeID(nil, string(types.EventTypeTransactionExecuted))
types.EVMLocation{}.TypeID(nil, string(types.EventTypeBlockExecuted))
```

Reply:
Good point :+1: , I will update them as soon as we add `flow-go` as a dependency. Right now it's not there yet, due to some dependency conflicts with `flow-go-sdk` I think. I have added a TODO and will update it though.